services/persistence/Dockerfile

```diff
@@ -1,15 +0,0 @@
-#FROM ubuntu:16.04
```

Comment:
ist die Idee, dass wir halt einfach die sqlite db in die entsprechenden container einhaengen? dann brauchen wir ja wirklich keinen persistence container mehr.